integrationtests/flag_builder_test.go

```diff
@@ -0,0 +1,131 @@
+//go:build integrationtests
```

Comment:
This is a whole new builder scheme just for these tests - we need a simple way to create flags via REST API. The existing methods were single-purpose.